templates/documentation/sdks/nocode/contentful.md

```diff
@@ -6,65 +6,62 @@ metadata:
+## Installing the api.video plugin
 
-1. A new JSON object
-2. Add it to the Component:Video Object
+Log into your Contentful account and add the [api.video app](https://www.contentful.com/marketplace/app/api-video/) by clicking Apps -> Marketplace
 
+![](/_assets/nocode/contentful/contentful-marketplace.png)
 
-### A new JSON object
+Select the Featured tab and search for the api.video plugin in the search bar
 
-In the top menu, select "Content Model," and a new page will open with all the models/components in your app. Choose the "Add content type" button on the top right to begin creating a new type:
+![](/_assets/nocode/contentful/contentful-search.png)
 
-1. The video field
-First, you will name your model, and then the UI will walk you through adding fields. Click the blue button to add a field and add a JSON Object. You'll be asked to name the field - choose a name that reminds you that it is an api.video object that is being added.
+Once you've found the api.video plugin, click on the icon. You will get a new page where you see the ability to install the plugin, click on Install and authorize the access for api.video
 
-Click the "Create and Configure" button, and under "Appearance," choose api.video.
+![](/_assets/nocode/contentful/contentful-auth.png)
 
-2. Name field:
- Create a new field, text, Short text, and when you configure the item, click the "Entry title" button. (If you do not add this step, every object you add will be named "Untitled," which is less than ideal.)
+Enter your [api.video API key](https://docs.api.video/reference/authentication-guide#retrieve-your-apivideo-api-key) an click Install
 
-### Modifying the Component:video model
+![](/_assets/nocode/contentful/contentful-apikey.png)
 
-Contentful has a built-in video model to which you can add the api.video app. Let's walk through the steps to do that.
+Congratulations! you have installed the api.video plugin successfully!
 
-In the "Content Model" section, click on the "Component:Video" model. This will give you a page that shows all the fields in the model. It has 2 fields, the title, and a YouTube Video Id by default. We'll modify this so that you can still use a YouTube video Id, but you can also add an api.video object.
+## Usage
 
-#### Add the api.video object
+After a successful installation, we can to the fun part of adding videos to our content. All you have to do is create a JSON object with the api.video video appearance type.
 
-Click the "add field" button on the right. Add a JSON object, and name the field "apivideo" or another term that helps you remember where the videos will be stored. Click the "Create and Configure" button, and then under Appearance, add the api.video app.
+### Adding api.video to the Content Model
 
+1. Create or edit an existing Content Model
+2. Add a new field with a JSON object type
+![](/_assets/nocode/contentful/contentful-select-object.png)
+3. Give the new JSON object a name and api key 
```

Comment:
`api key` you mean `field id` ?

Reply:
thanks for spotting this. Let's approve it and merge, I have another branch that is conflicting so I will fix it after this is mergedd

Reply:
Merged ✅